public/app/features/explore/Logs/LogsTable.tsx

```diff
@@ -74,36 +92,44 @@ export const LogsTable: React.FunctionComponent<Props> = (props) => {
         field.config = {
           ...field.config,
           custom: {
-            filterable: true,
             inspect: true,
+            filterable: true, // This sets the columns to be filterable
             ...field.config.custom,
           },
+          // This sets the individual field value as filterable
+          filterable: isFieldFilterable(field, logsFrame ?? undefined),
         };
       }
 
       return frameWithOverrides;
     },
-    [logsSortOrder, range, splitOpen, timeZone]
+    // We dont want to re-render whenever the splitOpen function changes, so it's being purposefully excluded from the deps
```

Comment:
Wouldn't that cause `getFieldLinksForExplore()` to hold a stale reference?

Reply:
Potentially, it's just that one dependency was responsible for more then 50% of the re-renders of this component. I added it back in as a dep tho, hopefully when these components get refactored we also clean up all the unnecessary re-rendering.